sass/components/_bulma.scss

```diff
@@ -50,8 +50,14 @@ $blue: rgb(19,129,161);
 $primary: $purple;
 $link: $purple;
 
-a:hover {
-    color: $purpleHover;
+html {
+  a {
+    color: $purple;
+    &:hover {
+        color: #363636;
```

Comment:
why is the hover color now hard coded to `#363636` instead of `$purpleHover` as it used to be?

Reply:
On https://staging.riffplatform.com/room, the $purpleHover variable is eventually being overwritten by #363636, so I assumed that what is seen on the platform is what I should do Edu.

Reply:
Interesting. OK, that makes sense for now, but I think some of this is going to need to be revisited.

Not because of your changes, but because it feels like styling pieces directly and w/ magic values has happened more that it should.

But keep in mind that your answer above is the kind of information that is useful to have as a comment in the code for the next person who is making changes (or you in 3 months when you've forgotten).